packages/dom/src/dom.js

```diff
@@ -60,14 +60,17 @@ function isSelectionForward( selection ) {
 }
 
 /**
- * Check whether the selection is horizontally at the edge of the container.
+ * Check whether the selection is at the edge of the container. Checks for
+ * horizontal position by default. Set `isVertical` to true to check only
+ * vertically.
  *
- * @param {Element} container Focusable element.
- * @param {boolean} isReverse Set to true to check left, false for right.
+ * @param {Element} container  Focusable element.
+ * @param {boolean} isReverse  Set to true to check left, false to check right.
+ * @param {boolean} isVertical Set to true to check only vertical position.
  *
  * @return {boolean} True if at the horizontal edge, false if not.
  */
-export function isHorizontalEdge( container, isReverse ) {
+function isEdge( container, { isReverse, isVertical } ) {
```

Comment:
How about renaming `isVertical` to `onlyVertical`? My brain was confused by this name and somehow assumed it worked the same as `isReverse` (check vertical or horizontal).

Reply:
Fixed in ccce7e0.